mozilla/foundation.mozilla.org
# mobile formatting of left/right images in body fields

Closes #7579 


**Link to sample test page**: https://foundation-s-article-im-bv65ld.mofostaging.net/en/publication-page-with-child-article-pages/fixed-title-article-page/

**Steps to test**:

1. Visit the test page linked above
2. Scroll down to take a look at the content 
3. The images should now maintain their positions and have the text wrap around them at all breakpoints.
4. If everything is behaving as expected, testing is complete!
